storage/range.go

```diff
@@ -392,7 +415,10 @@ func (r *Range) WaitForLeaderLease(t *testing.T) {
 // another node. It is false when a range has been created in response
 // to an incoming message but we are waiting for our initial snapshot.
 func (r *Range) isInitialized() bool {
-	return len(r.Desc().EndKey) > 0
+	if desc := r.Desc(); desc != nil && len(desc.EndKey) > 0 {
```

Comment:
remnant from running the whole shebang with leader lease on `start()`. Must be some test doing its initialization in a weird way. I'll undo this.